### DISCUSSION TITLE
求解，关于dnsmasq的一个小疑惑

### DISCUSSION THREAD

**Discussion by xy9861** [vernesong/OpenClash#4252] [Q&A]
我的设备结构是光猫下接了路由，我让连接光猫WiFi的设备网关改成路由wan地址，什么都是正常的，但是当我出于强迫症等小原因不让dnsmasq监听wan地址，光猫WiFi下的设备就不能解析dns了。此时明明可以访问路由lan口的其他服务，却不能用dns呢？有没有办法实现这一点

**Comment by fejeIaqbhbv** [comment]
额，这虽然用到dnsmasq，但不研究它，以后还是换地方问吧。

改WiFi下的设备DNS
